package.json

```diff
@@ -4,9 +4,9 @@
   "version": "1.0.0",
   "scripts": {
     "check": "mongodb-js-precommit",
-    "pretest": "mongodb-runner start",
+    "pretest": "mongodb-runner start --topology=replicaset",
     "test": "mocha",
-    "posttest": "mongodb-runner stop",
+    "posttest": "mongodb-runner stop --topology=replicaset",
```

Comment:
Please move `mongodb-runner stop --topology=replicaset` out of "pretest" and into the mocha `after()` hook, and likewise move the `mongodb-runner start` to the `before()` hook.

Reason: `posttest` never fires when the test fails. (Discussion: https://github.com/npm/npm/issues/5493)

The replicaset won't be cleaned up on failure. This is a problem for any CI environment, and is an anti-pattern we have repeated across several modules. Let's start fixing it.